frontend/src/lib/i18n/locales/en.json

```diff
@@ -26,6 +26,16 @@
       "remember": "Remember me",
       "forgot": "Forgot your password?",
       "sign_in_button": "Sign in"
+    },
+    "create_course": {
+      "errors": {
+        "name": "Course name can't be blank"
+      },
+      "title": "Create Course",
+      "input_label": "Enter course name",
+      "create_button": "Create Course",
```

Comment:
Let's be consistent and use same register for course everywhere.
I.e. if it's (Enter **course** name) than it should also be "Create **course**" (not Create **Course**), etc.
Downcase for course is ok there